### ISSUE TITLE
Page titles aren't announced when navigating on a screen reader

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#390]
Page titles must always be announced when you visit a new page. For some reason, it's inconsistent on which page title is announced.

Here is what I noticed:
- [ ] After logging in, there is no announcement that you're on the reports page
- [x] After clicking "Create Report" button, there is no announcement that the Create Report page is loaded.
- [x] After clicking "View Report" from the reports page, there is no announcement that you're on the report detail page.
- [x] After clicking a message in a report, there is no announcement that the Message Details page is loaded. 
- [x] After clicking a page in a report, there is no announcement that the Page Details page is loaded. 
- [x] After clicking "Delete Report" button, there is no announcement that the Reports page is loaded.
- [x] After clicking "Scans" from the navigation, the page title isn't read.
- [x] After clicking "Properties" from the navigation, the page title isn't read.
- [x] After clicking "Add Property", the new page title isn't read.
- [x] After clicking "Delete Property", the new page title isn't read.
- [x] After clicking "My Account" from the navigation, the page title isn't read.
- [x] After clicking "Logout" button, there is no announcement that you're now on the login page.
- [ ] After clicking "Delete Account" button, there is no announcement of the page title you are redirected to.
- [x] When logged out, if I visit https://equalify.dev/reports I am redirected to the signup page. The signup page title is not read.
- [ ] When logging in, reports page title isn't read.

Since this is a single page app, you might need to update the way the document title is added.

**Comment by wilsuriel03** [comment]
working on it

**Comment by wilsuriel03** [comment]
complete ✅

**Comment by bbertucc** [comment]
@kevinandrews1 this should be closed. The pages titles are not announcing on voice over. I checked off the items that are announcing, but most arent.

I created this video to show: https://www.loom.com/share/28b840331475403b94387eaae64bf3fd?sid=fd75ebbf-2865-4906-8ac9-299d0984e6a9

Reassigning this to @wilsuriel03

**Comment by bbertucc** [comment]
Looks like this update wasn't deployed yet, so we'll have to wait on testing it.

**Comment by bbertucc** [comment]
@kevinandrews1 can you test to make sure page titles are being read at http://equalify.dev ?

Once everything is good, I'll create a PR to merge with prod.

**Comment by kevinandrews1** [comment]
@bbertucc  / @wilsuriel03, I'm on Chrome+JAWS for this:
- After logging in, there is no announcement that you're on the reports page (reports title page isn't read).
- After clicking "Logout" button, there is no announcement that you're now on the login page.
- After clicking "Delete Account" button, there is no announcement of the page title you are redirected to.

Comments on some of the others:
- I couldn't test messages or pages when viewing the test report with Blake's login--for some reason, no results. Can one of you confirm the page title is read out for you for both?
- Delete report or property does read the dialog properly and focus lands on the cancel button. JAWS does read reports page loaded if it's report for example after you delete a report. You're not notified about the action that the report was deleted but that's a separate issue. (Sorry Blake, I deleted the test report while testing but you can always recreate it).

**Comment by kevinandrews1** [comment]
To clarify this was where I still ran into problems. The others are good to go @bbertucc @wilsuriel03.

**Comment by bbertucc** [comment]
@wilsuriel03 can you check on those three issues that Kevin reported? Those seem to be the only remaining issues.

Re-assigning this to you. Ping me here with any updates or questions! Thanks!

**Comment by wilsuriel03** [comment]
ready for testing @bbertucc
